pkg/ndm-grpc/server/info.go

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
maybe good to move this to root directory "api-service" 

The content under "server" may be moved to "api-service/node" - indicating these are the services offered by per node (daemonset). 

Going forward the cluster level services can be added under "api-service/cluster".

Reply:
About the naming convention, underscores and spaces aren't allowed so that leaves us with only hyphens. Couldn't find anything about cases though.